projects/github.com/bcicen/ctop/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://github.com/bcicen/ctop/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: ctop
+
+versions:
+  github: bcicen/ctop
+
+build:
+  dependencies:
+    go.dev: ^1.18
+  script: |
+    go mod download
+    go build -v -trimpath -ldflags="$LDFLAGS" -o $BUILDLOC
```

Comment:
in general, i've been using `GO_LDFLAGS`, in case go triggers a C-compiler which gets confused by weird `LDFLAGS` variables.

Reply:
✅